packages/flutter/test/material/floating_action_button_test.dart

```diff
@@ -20,8 +20,11 @@ import 'feedback_tester.dart';
 
 void main() {
 
-  final ThemeData material3Theme = ThemeData.light().copyWith(useMaterial3: true);
-  final ThemeData material2Theme = ThemeData.light().copyWith(useMaterial3: false);
+  //final ThemeData material3Theme = ThemeData.light().copyWith(useMaterial3: true);
+  //final ThemeData material2Theme = ThemeData.light().copyWith(useMaterial3: false);
```

Comment:
nit: seems we can remove these two lines.

Reply:
Oops, good catch, thanks.